prow/Makefile

```diff
@@ -11,6 +11,19 @@ PROJECT_BUILD ?= istio-prow-build
 ZONE    ?= us-west1-a
 CLUSTER ?= prow
 
+generate-istio-private-jobs:
+	go run ./genjobs \
+	--clean \
```

Comment:
I'd indent this deeper than the `go run` so its clear its part of the `go run` command.